### PR TITLE
add log into user controller

### PR DESCRIPTION
**Pull request by HGChamara**

### CODE REVIEW

File: Rest_webService/src/main/java/com/web/rest/controller/UserRegistrationController.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	@ResponseBody
 	public UserRegisterResponse registerUser(@RequestBody UserRegisterRequest userRegistration) 
 	{
+		System.out,println("inside registerUser()");
```
Comment (putchar91) [inline Rest_webService/src/main/java/com/web/rest/controller/UserRegistrationController.java:19]: why no log

**Review by putchar91 — CHANGES_REQUESTED** [review]

File: Rest_webService/src/main/java/com/web/rest/controller/UserRegistrationController.java (hunk 1 above)
Comment (HGChamara) [reply Rest_webService/src/main/java/com/web/rest/controller/UserRegistrationController.java:19]: no log libs integrated

**Comment by putchar91** [comment]
ok

**Review by putchar91 — APPROVED** [review]
ok approve

**Pull request merged**